dev/stacks/dpe-sandbox-k8s-deployments/variables.tf

```diff
@@ -13,6 +13,11 @@ variable "node_security_group_id" {
   type        = string
 }
 
+variable "pod_to_node_dns_sg_id" {

```

Comment:
No. We would have no way of knowing what it would be. The way this works is in the spacelift terraform modules you'll see this block:
```
resource "spacelift_stack_dependency_reference" "pod-to-node-security-group-id-reference" {
  stack_dependency_id = spacelift_stack_dependency.k8s-stack-to-deployments.id
  output_name         = "pod_to_node_dns_sg_id"
  input_name          = "TF_VAR_pod_to_node_dns_sg_id"
}
```

This is passing along the output from the k8s infrastructure (Which contains the EKS module) as input to the k8s deployments stack (Which deploys the kubernetes resources).